.github/ISSUE_TEMPLATE/patch_release.md

```diff
@@ -22,27 +22,31 @@ this issue for posterity. Assuming `vX.Y.Z-up.K` is being cut, after upstream
 [crossplane/crossplane][upstream-xp] `vX.Y.Z` has been released
 according to the declared schedule, you should have:
 
-- [ ] Synced the `release-vX.Y` release branch in [upbound/crossplane][upbound-xp-fork], with upstream [crossplane/crossplane][upstream-xp] release branch, up to the `vX.Y.Z` tag, adding any required change specific to the fork, see [here][sync-xp-fork] for more details.
+- [ ] Synced the `release-X.Y` release branch in [upbound/crossplane][upbound-xp-fork], with upstream [crossplane/crossplane][upstream-xp] release branch, up to the `vX.Y.Z` tag, adding any required change specific to the fork, see [here][sync-xp-fork] for more details.
 - [ ] Tagged [upbound/crossplane][upbound-xp-fork] `vX.Y.Z-up.K` from the `release-X.Y` branch by:
```

Comment:
Maybe we could add a warning to make sure that we are in the correct repo? I can remember I've ran a workflow on the wrong repo once.  Similar warnings could be for other steps.
Something like:

> [!WARNING]
> 
> Please make sure that you are in the [upbound/crossplane][upbound-xp-fork] repo before running the workflows.

Reply:
Every step would need a warning most probably 😅 we have to try to make it succinct but informative enough, hopefully automating it if possible down the line

Reply:
Actually! We could switch a lot of click steps to exact commands using `gh`! I know you are a fan of gh too, would you be interested into leading on that?

Reply:
Yes! That's a great idea, the less we require from the release lead type, the better!